homeassistant/components/here_travel_time/__init__.py

```diff
@@ -38,6 +41,23 @@ async def _async_update_at_start(_: HomeAssistant) -> None:
     return True
 
 
+def alert_for_multiple_entries(hass: HomeAssistant) -> None:
+    """Check if there are multiple entries for the same API key."""
+    if len(hass.config_entries.async_entries(DOMAIN)) > 1:
+        async_create_issue(
+            hass,
+            DOMAIN,
+            "multiple_here_travel_time_entries",
+            learn_more_url="https://github.com/home-assistant/home-assistant.io/pull/39653",
```

Comment:
I don't like to point it to a PR. Did here announce it publicly on their website/blog somewhere? So we could use that URL instead.
Otherwise I suggest to point to the docs instead a github PR

Reply:
On their official blog HERE posted that new customers will not have access to the previous "HERE Limited Plan" and stated that existing customers are not affected: https://www.here.com/learn/blog/april-2025-platform-release-notes

However, existing customers started to receive emails that they have to migrate. A picture of that email is in the PR https://github.com/home-assistant/home-assistant.io/pull/39653.

An official public communication of the content of these emails does not exist.

I will therefore link to the Home Assistant docs.